src/Components/PetCard.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import '../Styles/PetCard.scss';
 
-
-const PetCard = () => {
+const PetCard = ({ id, name }) => {
```

Comment:
I really appreciate the use of this in-line styling in order to remove the text decoration of the link.